rocketpool/api/pdao/replace-security.go

```diff
@@ -35,6 +37,37 @@ func canProposeReplaceMemberOfSecurityCouncil(c *cli.Context, existingMemberAddr
 	response := api.PDAOCanProposeReplaceMemberOfSecurityCouncilResponse{}
 
 	// Get node account
+	nodeAccount, err := w.GetNodeAccount()
+	if err != nil {
+		return nil, err
+	}
+
+	// Sync
+	var isRplLockingAllowed bool
+	var wg errgroup.Group
+
+	// Get is RPL locking allowed
+	wg.Go(func() error {
```

Comment:
You usually want to use goroutines when starting multiple queries at the same time. If you have only one query then you don't need to use threads.

Reply:
Thanks for the tip! I've fixed the usage of goroutines.